.golangci.yml

```diff
@@ -188,6 +188,8 @@ issues:
   # We are disabling default golangci exclusions because we want to help reviewers to focus on reviewing the most relevant
```

Comment:
Bumping dependencies embedded in the code is unrelated to what we use for testing

Reply:
Yes and no. We use the API types from our dependencies to deploy stuff like VirtualMachine etc.

But if we're sure it's not a problem we can do this in a follow-up (but I think we should follow-up).

I mostly got confused because the VirtualMachineImage was cluster scoped in v1.8.5 and then I started looking. But I think this also didn't change with v1.8.6..

Reply:
In v1.8.6 there's v1alpha2 and things changed.

Before there was a cluster-scoped and non-cluster-scoped `VirtualMachineImage`.

Now there is a cluster-scoped `ClusterVirtualMachineImage` if I got it right, and the non-cluster-scoped `VirtualMachineImage`.

Reply:
Let's just do a follow-up to bump go dependencies as well before v1.10.0